README.md

```diff
@@ -125,6 +146,10 @@ nix-shell --run ./scripts/buildkite/check-stylish.sh
 ```
 
 ## Tests
+<details>
+<summary>
+Consensus test components.
+</summary>
```

Comment:
I like this!

I was recently thinking: is this list of commands even useful to have here, I'm not sure we even maintain it. Personally, I wouldn't mind if it were deleted, but others might disagree.

Reply:
I don't mind removing both lists.

Reply:
I don't think anybody of us is regularly using `nix-build`, it's quite likely that things has changed since these commands where written in the `README.md` file.

Reply:
Yep, I agree. This seems especially sound if running these via `cabal` inside of `nix-shell` is close enough to indistinguishable from running them via `nix-build`. I would think so, except maybe for things like what kind of file handle `stdin` is (termtype etc). Likely fine.